components/screens/Match.tsx

```diff
@@ -1,7 +1,8 @@
 import React, { FunctionComponent, useEffect, useState } from 'react'
 import { StyleSheet, Text, View, TouchableOpacity } from 'react-native'
 import { RouteProp } from '@react-navigation/native'
-import { createDrawerNavigator } from '@react-navigation/drawer';
+// import { createDrawerNavigator } from '@react-navigation/drawer';
```

Comment:
@franchizzle had to take out the drawer functionality because it broke in Expo 49. We will have to come back and fix this later if we still want to use the react-navigation Drawer component

Reply:
@jjPlusPlus I'm sure we can find an alternative / make our own. Don't think we need exactly that.